src/main/java/com/weirddev/testme/intellij/template/context/Method.java

```diff
@@ -152,4 +158,13 @@ public boolean hasParams() {
         return !methodParams.isEmpty();
     }
 
+    /**
+     *
+     * true - if method has Exception
+     */
+    public boolean hasException(){
+        return StringUtils.isNotEmpty(methodExceptionTypes);
```

Comment:
think we can do without this method. better simplify and init field to null if resolved as empty string